Sources/OktaLogger/okta-logger-swift.swift

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
Why we need this file?

Reply:
Funny reason. Due to our modularity (a target for instabug, firebase, core etc), the "complete" version imports all of the targets but doesn't have any source files in it. And SPM complains that there are no sources for the target (besides the dependencies) so I added this file as a workaround. Maybe it would make more sense to name it `dummy.swift`?

Reply:
Ok, I see. As I know cocoapods also adding dummy files to framework targets
How about to name it: spm_dummy.swift?